### ISSUE TITLE
Error with reverb installation

### ISSUE THREAD

**Issue by samarjeet** [deepmind/acme#2]
Thanks for making this project publicly available. I am checking it out. reverb installation is hitting the following error 

_ERROR: Could not find a version that satisfies the requirement dm-reverb-nightly==0.1.0.dev20200529; extra == "reverb" (from dm-acme[reverb]) (from versions: none)
ERROR: No matching distribution found for dm-reverb-nightly==0.1.0.dev20200529; extra == "reverb" (from dm-acme[reverb])_

**Comment by fastturtle** [comment]
Hi samarjeet, thanks for trying out Acme! Are you installing this on a Linux based OS (Reverb only supports Linux based OSes)?

If you don’t have access to a Linux OS you can always use docker to try out Acme!

**Comment by samarjeet** [comment]
Yeah, I am trying it on WSL which for all practical purposes "quacks" like Ubuntu.

**Comment by fastturtle** [comment]
Just to rule out versioning issues, which version of Python are you using?
